sass/cd-header/_cd-nav.scss

```diff
@@ -0,0 +1,375 @@
+/**
+ * Common Design Navigation in header.
+ */
+
+.cd-site-header__nav-holder {
+  float: left;
+}
+
+.cd-site-header__nav-toggle {
+  background: transparent;
+  border: 0;
+  border-left: 1px solid $cd-border-color;
+  border-right: 1px solid $cd-border-color;
+  font-size: map-get($cd-font-sizes, medium);
+  font-weight: bold;
+  color: $cd-default-text-color;
+  height: $cd-site-header-height;
+  padding: 0 20px;
+  text-transform: uppercase;
+  transition: background 0.3s ease;
+
+  .icon-arrow-down {
+    margin-left: 2px;
+    color: $cd-highlight-red;
+  }
+
+  .open & {
+    position: relative;
+    z-index: 101;
+    color: $cd-dark-text-color;
+
+    &:after {
+      bottom: -3px;
+      content: '';
+      background: $cd-highlight-red;
+      height: 3px;
+      left: -1px;
+      position: absolute;
+      right: -1px;
+      transition: background 0.3s ease;
+      width: calc(100% + 1px);
+    }
+
+    &:focus {
+      background: $cd-white-bg-color;
+    }
+  }
+
+  &:hover,
+  &:focus {
+    color: $cd-dark-text-color;
+    background: $cd-mid-bg-color;
+    outline: none; // default browser outline is replaced by background colour change.
+  }
+}
+
+.cd-nav {
+  background: $cd-white-bg-color;
+  height: calc(100vh - (#{$cd-global-header-height} + #{$cd-site-header-height} + 3px));
+  padding: 20px 0 36px 0;
+  position: absolute;
+  right: 0;
+  left: 0;
+  top: calc(#{$cd-global-header-height} + #{$cd-site-header-height} + 3px);
+  width: 100vw;
+  z-index: 100; // arbitrary high zindex, may need adjusting depending on other site content.
+
+  ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  a,
+  button {
+    color: $cd-default-text-color;
+    display: block;
+    padding: 15px 30px;
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+      background: $cd-mid-bg-color;
+      color: $cd-dark-text-color;
+      outline: none; // default browser outline is replaced by background colour change.
+    }
+  }
+
+  button {
+    background: none;
+    border: 0;
+    font-size: map-get($cd-font-sizes, default);
+    text-align: left;
+    width: 100%;
+  }
+}
+
+.cd_nav > ul > li,
```

Comment:
This is the Drupal-menu equivalent (without Drupal's own menu classes)

Reply:
Just a thought, here. If we want to avoid re-implementing Drupalisms in our CSS, we could actually throw whole chunks of the core CSS out using a pretty simple technique in the `ocha_basic.info` file:

```
stylesheets[all][] = unset/NAME.css
```

Where each `NAME` is the filename of the core stylesheet. The `unset` directory need not exist; it provides the cue that the CSS is being unset and could be named anything at all.

Reply:
Nice. I didn't think of that. I'll see if I can do so for the main menu so we don't get the weird .leaf styling